src/sidebar/components/slider.js

```diff
@@ -2,6 +2,12 @@ import { createElement } from 'preact';
 import { useCallback, useEffect, useRef, useState } from 'preact/hooks';
 import propTypes from 'prop-types';
 
+/**
+ * @typedef SliderProps
+ * @prop {Object} [children]
+ * @prop {boolean} [visible] - Whether the content should be visible or not.
```

Comment:
```suggestion
 * @prop {boolean} visible - Whether the content should be visible or not.
```

It doesn't make sense to use `Slider` without the `visible` prop.